pilot/test/integration/driver.go

```diff
@@ -219,7 +219,7 @@ func runTests(envs ...infra) {
 			&routingToEgress{infra: &istio},
 			&zipkin{infra: &istio},
 			&authExclusion{infra: &istio},
-			&kubernetesExternalNameServices{infra: &istio},
+			//&kubernetesExternalNameServices{infra: &istio},
```

Comment:
Please a a comment with the issue (if there is one) and explaining it's removed until the issue is fixed.

Reply:
According to https://github.com/istio/istio/pull/3039#issuecomment-362159173:
> There is a general problem affecting also http and tcp tests, there is no point in disabling.

The PR has do-not-merge/hold label. I guess the idea currently is to investigate the issue first. The issue is https://github.com/istio/istio/issues/3037.